index.php

```diff
@@ -659,7 +659,49 @@ static function forumNewExtern()
 }
 else
 {
+	
 	print '<div class="content-bare content-home-header">';// content-follow-on">';
+	
+	// home page tutorial
+	if (isset($_COOKIE['wD-Tutorial-Index'])) 
+	{
+		$tutorialMessage = l_t(
+			'To help you become familiar with webDiplomacy, we prepared a few brief presentations
+			for you that you will find on different pages of the site. You will only see these once apiece, but if you
+			ever want to review what you read, you can check out our <a href="faq.php" target="_blank">Frequently Asked Questions</a>. 

```

Comment:
Can we make a button at the top of each page with these with a ? icon that will reset the cookie for that page so someone can re-review these if they want?

Reply:
We talked on this and this will be a follow up PR.